components/browser/engine-gecko/src/main/java/mozilla/components/browser/engine/gecko/ext/GeckoChoice.kt

```diff
@@ -0,0 +1,19 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.browser.engine.gecko.ext
+
+import mozilla.components.browser.engine.gecko.prompt.GeckoChoice
+import mozilla.components.concept.engine.prompt.Choice
+
+/**
+ * Converts a GeckoView [GeckoChoice] to an Android Components [Choice].
+ */
+fun GeckoChoice.toChoice(): Choice {
+    val choiceChildren = items?.map { it.toChoice() }?.toTypedArray()
+    // On the GeckoView docs states that label is a @NonNull, but on run-time
```

Comment:
Could we file a bug on [bugzilla](https://bugzilla.mozilla.org/enter_bug.cgi?product=GeckoView) for make the aware? Also we could link here :)

Reply:
Thanks you!
Could we add it as part of the comment?